src/pages/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./LinkList";
```

Comment:
I have no strong opinion and we do not have a tendency to use `index` files or not use them.

In general, I see them as too verbose, but as I said, it is not a strong opinion.

@jcamposmk, @Cromeror, do you have an opinion about using these index files by default?

Reply:
I have no strong opinion about this ...

Reply:
It's called Barrels.
This technique is widely used. It can be used or not. In general, I like to use it. certain plugins generate and maintain them automatically.

Reply:
I remove these types of usage whenever I notice it in my code, but I can live with it. I like show the code with less jump in the traceability to found the source code of component.